Riot/Modules/SideMenu/SideMenuCoordinator.swift

```diff
@@ -287,3 +291,28 @@ extension SideMenuCoordinator: SpaceListCoordinatorDelegate {
         self.showMenu(forSpaceWithId: spaceId, from: sourceView)
     }
 }
+
+// MARK: - SpaceMenuPresenterDelegate
+extension SideMenuCoordinator: SpaceMenuPresenterDelegate {
+    func spaceMenuPresenter(_ presenter: SpaceMenuPresenter, didCompleteWith action: SpaceMenuPresenter.Actions, forSpaceWithId spaceId: String, with session: MXSession) {
+        presenter.dismiss(animated: false) {
+            switch action {
+            case .exploreRooms:
+                self.exploreRoomCoordimator = ExploreRoomCoordinatorBridgePresenter(session: session, spaceId: spaceId)
```

Comment:
`CoordinatorBridgePresenter`s are designed to be used only in ObjC they have been made only for backward compatibility, we should use Coordinators directly in Swift.